creme/stats/iqr.py

```diff
@@ -34,13 +36,19 @@ class IQR(base.Univariate):
             400
             450
             500
+
     """
 
     def __init__(self, q_inf=0.25, q_sup=0.75):
         if q_inf >= q_sup:
             raise ValueError('q_inf must be strictly less than q_sup')
-        self.quantile_inf = quantile.Quantile(quantile=q_inf)
-        self.quantile_sup = quantile.Quantile(quantile=q_sup)
+        self.q_inf = q_inf
+        self.q_sup = q_sup
+        self.quantile_inf = quantile.Quantile(quantile=self.q_inf)
+        self.quantile_sup = quantile.Quantile(quantile=self.q_sup)
+
+    def __str__(self):
+        return f'rolling_{self.__class__.__name__}_{self.q_inf}_{self.q_sup}'
```

Comment:
I think you're inversed the `__str__` method of both classes. Also, you type `iqr` instead of `self.__class__.__name__`.

Reply:
I'll merge the PR and take care of this, it's no biggie :)